index.rst

```diff
@@ -156,11 +130,6 @@ or before/after major
 activities. After each release, user will be asked to review and possibly remove notebooks older than 1 year to make
 sure stale notebooks are not lingering alongside the main working branch.
 
-.. note::
-
-    Another suggestion is to use git-lfs for large files in the repo. This needs further
-    exploration. A copy would need to be available at the summit should the network go down.
-
 
 .. _Observing_Utilities:
 
```

Comment:
I think the first paragraph could be broken down into two:

> Observing utilities are user-defined methods that perform tasks that are **still** not ~already~ part of the **base** control packages ~code base that operates the observatory~. The Control Packages section discusses this in further detail. 
>
> An example of functionality contained in a utility would be the reduction/analysis of an image. In the use-case discussed in this document, the user defines methods that perform basic ISR on an image, finds the center of the star, and calculates the required offset. In the cases where image reduction and/or analysis is required, specifically for ComCam and LSSTCamera images, the processing ~will~ **may** utilize the OCS Controlled Pipeline Service (OCPS), which is still undergoing design and development. More details on it’s use during development will be added once available.

Reply:
Saying "still not part of" says we already had this on our list of things to do but we haven't done them yet.... which is not the case here.
I did modify the phrase to use "base control package" which is a better term.